developer-workflow/lang-changes.rst

```diff
@@ -41,25 +47,104 @@ Council can truly classify something as Pythonic, you can read the
+  It's good to use the PEP template as a guide for what information to include,
+  but don't try to write a full PEP until the idea has been accepted for
+  further consideration.
+
+- **Don't** create a list of proposed changes. Each idea needs its own
+  discussion. Proposing a change is significant work. You will need to spend
+  time on each proposal. A tossed-off list will not be taken seriously.
+
+When justifying your idea, these are not good reasons to make a change:
+
+- "Other languages work this way." Languages are different. What works well
+  in one language may not work well in Python, or Python may already have a
+  way to do a similar thing.
+
+- "It would be nice." This is too vague. Be specific about the benefits
+  your change will bring, and consider the costs as well.
+
+Some things you **should** do:
+
+- Research whether your idea has been proposed before. There are many
+  suggestions that have been made and rejected in the past. If your idea has a
+  history, read the discussion to see why it was rejected. Previously rejected
+  ideas are especially unlikely to get accepted. If you can address the
+  concerns raised in the previous discussion, mention that in your proposal.
+
+- Read other ideas to see how they are presented. This will help you flesh out
+  your proposal.
+
+- Read other ideas to understand the kinds of concerns that are raised and
+  objections that must be answered. Your proposal will be stronger if you can
+  address these concerns up front.
+
+- Provide enough detail to fully explain your idea, and to show that you have
+  considered all the implications of the change.  Strike the right balance
+  between brevity and completeness.  A shorter proposal will be easier to
+  discuss, so long as it includes all the necessary information.
+
+
+Engage in discussion
+^^^^^^^^^^^^^^^^^^^^
+
+Once you post your proposal, people will respond. You need to continue
+the discussion, answer questions, and address objections. This is an important
+part of the process. Other people's perspectives will help explore the full
+impact of the idea and find strengths or weaknesses you may not have
+considered.
+
+You will almost always get at least some push-back. Don't be disappointed,
+and don't take it personally.
+As the long list of :pep:`Withdrawn and Rejected PEPs
+<0#rejected-superseded-and-withdrawn-peps>` in the :pep:`PEP Index <0>`
+attests, and as befits a mature programming language, getting significant
+changes into Python isn't a simple task.
+
+Engage in good faith. The goal of the discussion is to find the best balance
+between competing concerns. If your idea has merit, the discussion will help
+refine it and make it stronger. If your idea has flaws, the discussion will
+help identify them so you can address them or move on. Keep the discussion
+productive and focused on the issues, not on personalities.
+
+It's especially useful to get core team members involved in the discussion.
```

Comment:
Can this be made more specific? Like

> It is especially useful to discuss with core team members because
> they know the language and boundary conditions very well,
> and eventually you'll need a core team member to sponsor a
> potential PEP.

"get core team members involved" may be misunderstood as actively asking. I've seen pings to core team members to get attention, but that's certianly not what we want to encourage.

Reply:
I added similar language.